newyork.html

```diff
@@ -20,12 +20,23 @@
         <label for="toggle">&#9776;</label>
```

Comment:
## Name change of file suggestion.
- [ ] What do you think about my proposal to rename this file to `index.html`?